docs/getting-started/getting-started-with-cwl.rst

```diff
@@ -99,7 +99,7 @@ repository:
         foaf: http://xmlns.com/foaf/0.1/
 
 .. note:: The `sbg:draft-2 <https://docs.sevenbridges.com/docs/about-the-common-workflow-language#sbgdraft-2>`__ implementation of CWL is optimized for the Seven Bridges cloud-based platform and includes custom extensions.
-    Although Dockstore does not support CWL sbg:draft-2, we do support CWL v1.0 which natively supports these custom extensions. Seven Bridges also provides `instructions <https://sb-biodatacatalyst.readme.io/docs/maintaining-and-versioning-cwl-on-external-tool-repositories>`__
+    Dockstore does not support sbg:draft-2 CWL tools and workflows and cannot be used to launch them on the Seven Bridges platform.  However, we do support `CWL v1.0 <https://www.commonwl.org/v1.0/>`__, which defines similar functionality and supersedes the sbg:draft-2 extensions.  Seven Bridges also provides `instructions <https://sb-biodatacatalyst.readme.io/docs/maintaining-and-versioning-cwl-on-external-tool-repositories>`__
```

Comment:
It's still a little fuzzy to me what "does not support" means.

Can I import a such a workflow/tool to Dockstore? If so, will it show up without errors in the UI? Can I run it via the CLI?

Does "cannot be used to launch them on the SB platform" mean the Launch with buttons are disabled, or does it mean they are enabled and the operation fails? I would say whichever it is.

Reply:
I made an edit to convey that if you register a sbg:draft-2 CWL, Dockstore will mark it as invalid, and you won't be able to publish, run, or launch it.  I'm wary of describing exactly how the UI will respond to unsupported activity, because it takes up a lot of space and creates an opportunity for the docs and product to get out of sync, if/when we change the behavior.